pkg/api/unversioned/well_known_taints.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unversioned
+
+const (
+	// when node down or unreachable
+	TaintNodeDown = "taint.alpha.kubernetes.io/nodedown"
```

Comment:
I think `node.alpha.kubernetes.io/nodeNotReady` would be better

(And please update all the other places where you wrote 'nodedown' to instead say nodeNotReady)